routers/users.py

```
@@ -0,0 +1,136 @@
+
+# 이미지 저장 경로 설정
+UPLOAD_DIR = "static/uploads"
+os.makedirs(UPLOAD_DIR, exist_ok=True)
+
+
+@router.post("/signup", status_code=201, response_model=SignupResponse)
+async def signup(
+        email: str = Form(...),
+        password: str = Form(...),
+        nickname: str = Form(...),
+        profileImage: Optional[UploadFile] = File(None)
```

Ruff (0.14.13)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)